dev/config.yml

```diff
@@ -15,7 +15,17 @@ playout:
 liquidsoap:
   server_listen_address: 0.0.0.0
 
+  harbor_ssl_certificate: /certs/fake.crt
+  harbor_ssl_private_key: /certs/fake.key
+
 stream:
+  inputs:
+    main:
+      public_url: https://localhost:8001/main
+      mount: main
+      port: 8001
+      secure: true
```

Comment:
According to #1032 we talked about adding a `secure_port` field to be able have both a secure and insecure port, but this requires to add an extra input harbor and require more handling, for instance how does the input behave if both input port are used at the same time.

Only allowing for a single input makes things simpler for the time being, and maybe we want to allow a list of inputs in the future to allow more advanced use cases.

So this adds a simple way to secure the input stream, until we draft a proper spec on how we want to handle more than 1 input stream for both main and show inputs.

My only question is whether the secure field is future proof, and if we will have to change it again when adding extra features for the input streams ?

Reply:
I don't think we'll have to change it again. I agree that a single input is much easier, and makes it a binary choice - are you using SSL for your LT deployment or not. We could change the result of the flag if needed later, but I can't foresee needing to do that at the moment